pkg/ccl/cliccl/load.go

```diff
@@ -169,7 +169,7 @@ func runLoadShow(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
-	desc, err := sqlccl.ReadBackupDescriptorFromURI(ctx, basepath)
+	desc, err := sqlccl.ReadBackupDescriptorFromURI(ctx, basepath, nil /* cluster.Settings */)
```

Comment:
here+elsewhere: if you felt so inclined, I'd love to see a alias like `settings.NoSetttingsValues = nil` so we could just use that in place of the commented nil args`nil /* ...*/`

Reply:
Done.